acceptance-tests/README.md

```diff
@@ -1,10 +1,10 @@
 # HubSpot CLI Acceptance Tests
 
-This project tests `hs.*` commands as if they were being used by an end-user.
+This project tests our CLI's `hs <*>` commands as if they were being used by an end-user.
 
 ## Getting Started
 
-The main test execution is kicked off by running `yarn test-cli` from the root of `hubspot-cli`. This will run the `run-tests` scipe which itself is a CLI, so to see the available options, you can `run-tests --help`. This was done so that the tests can be installed against any given version of the CLI and tested against it. In addition, it was done to keep the test logic separated from the CLI library.
+The main test execution is kicked off by running `yarn test-cli` from the root of `hubspot-cli`. This will run the `run-tests` scipt which itself is a CLI, so to see the available options, you can `run-tests --help`. This was done so that the tests can be installed against any given version of the CLI and tested against it. In addition, it was done to keep the test logic separated from the CLI library.
```

Comment:
A step in the right direction but I think this should be `script` haha

Reply:
😆

Reply:
so close lol